test_framework/scripts/longhorn-setup.sh

```diff
@@ -373,13 +399,20 @@ run_longhorn_tests(){
 
 main(){
   set_kubeconfig_envvar ${TF_VAR_arch} ${TF_VAR_tf_workspace}
-  create_longhorn_namespace
-  install_backupstores
+
   # set debugging mode off to avoid leaking aws secrets to the logs.
   # DON'T REMOVE!
   set +x
   create_aws_secret
   set -x
+
+  if [[ ${TF_VAR_k8s_distro_name} == "eks" ]]; then
+    create_admin_service_account
+    install_iscsi
```

Comment:
Do we need to install nfs/cifs client as well?

Reply:
Seems not necessary. Longhorn successfully installed without apply these two.